lib/tasks/evm_test_helper.rb

```diff
@@ -1,5 +1,6 @@
 module EvmTestHelper
-  VMDB_SPECS        = FileList["spec/**/*_spec.rb"].exclude(/^spec\/(replication|migrations|automation)/)
+  #TODO: Reuse the array of excluded directories for building the FileList exclusion
+  VMDB_EXCLUDED_SPEC_DIRECTORIES = %w(replication migrations automation).freeze
```

Comment:
A lot of this was from before we had rspec tagging.  Can we just leverage the tagging now?